test/extended/testdata/testjob-plugin.xml

```diff
@@ -53,16 +69,62 @@
       <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftServiceVerifier>
 
+    <!-- 1 to 1 mapping -->
```

Comment:
And again, a separate test job xml file that just includes what is needed for the multi tag test